fuse/data/ops/ops_debug.py

```diff
@@ -0,0 +1,125 @@
+from abc import abstractmethod
+from typing import Hashable, List, Sequence, Optional
+from fuse.data.utils.sample import get_sample_id
+from fuse.utils import NDict
+from fuse.data import OpBase
+import numpy
+import torch
+
+
+class OpDebugBase(OpBase):
+    """
+    Base class for debug operations.
+    Provides the ability to limit the samples to the debug.
```

Comment:
-> to limit samples to debug

Reply:
maybe mention that first K samples will be debugged and it will be passthrough for the rest?